Mathlib/Topology/Algebra/Order/Module.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2025 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Order.Nonneg.Module
+import Mathlib.Topology.Algebra.ConstMulAction
+
+/-!
+# Continuous nonnegative scalar multiplication
+-/
+
+variable {R α : Type*} [Semiring R] [PartialOrder R] [SMul R α] [TopologicalSpace α]
+  [ContinuousConstSMul R α]
+
+instance : ContinuousConstSMul {r : R // 0 ≤ r} α where
```

Comment:
Do we have the `ContinuousSMul` instance? If no, then could you please add it?

P.S.: Should [Nonneg.instSMul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Nonneg/Module.html#Nonneg.instSMul) take `[Zero R]` instead of `[Semiring R]`? [out of scope for this PR]

Reply:
I don't have any use at this time for weakening to `[Zero R]`